src/Card/Card.scss

```diff
@@ -206,6 +232,16 @@
   &.vertical {
     max-height: $card-image-vertical-max-height;
   }
+
+  .pgn__card-image-cap-loader {
+    .react-loading-skeleton {
+      margin-bottom: 1.25rem;
+      position: relative;
+      top: -.313rem;
+      min-width: 17.875rem;
```

Comment:
[curious] Why was this min-width necessary / what is it's purpose?

Reply:
I assumed that the elements of the skeleton will have the dimensions that the consumer will have, but not less than the minimum that I set. I like your idea of flexible consumer sizing